packages/cosmic-swingset/src/anylogger-agd.js

```diff
@@ -0,0 +1,165 @@
+/* eslint-env node */
+import {
+  getEnvironmentOptionsList,
+  getEnvironmentOption,
+} from '@endo/env-options';
+import anylogger from 'anylogger';
+import { defineName } from '@agoric/internal/src/js-utils.js';
+import util from 'util';
+
+/** @import {BaseLevels} from 'anylogger'; */
+/** @typedef {keyof BaseLevels} LogLevel; */
+
+const VAT_LOGGER_PREFIXES = Object.freeze([
+  'SwingSet:vat',
+  'SwingSet:ls', // "ls" for "liveslots"
+]);
+
+//#region Golang zerolog-like output affordances
+
+let shortDateTimeFormatter;
+const plainTime = d => {
+  if (!shortDateTimeFormatter) {
+    shortDateTimeFormatter = new Intl.DateTimeFormat(undefined, {
+      timeStyle: 'short',
+    });
+  }
+  const shortTime = shortDateTimeFormatter.format(d).replace(/(\d) /, '$1');
+  return shortTime;
+};
+
+const levelToPlainLevel = {
+  error: 'ERR',
+  warn: 'WRN',
+  info: 'INF',
+  log: 'LOG',
+  debug: 'DBG',
+};
+
+/**
+ * @param {Date} d
+ * @returns {string}
+ */
+const dateToNumericOffsetString = d => {
+  // TODO: Use Temporal.ZonedDateTime
+  // `toString({ fractionalSecondDigits: 0 })`.`
+  const pad2 = n => (n < 10 ? `0${n}` : `${n}`);
+  const utcOffsetMinutes = -d.getTimezoneOffset();
+  const utcOffsetAbsMinutes = Math.abs(utcOffsetMinutes);
+  const utcOffsetString = `${utcOffsetMinutes < 0 ? '-' : '+'}${pad2(Math.floor(utcOffsetAbsMinutes / 60))}:${pad2(utcOffsetAbsMinutes % 60)}`;
```

Comment:
I had to stare at that line for a while before I got it.

```suggestion
  const [sign, mm, ss] = [utcOffsetMinutes < 0 ? '-' : '+', Math.floor(utcOffsetAbsMinutes / 60), utcOffsetAbsMinutes % 60]
  const utcOffsetString = `${sign}${pad2(mm)}${pad2(ss)}`;
```

or perhaps a comment with a couple examples:

```suggestion
  // UTC offset e.g. -05:00 or +11:30
  const utcOffsetString = `${utcOffsetMinutes < 0 ? '-' : '+'}${pad2(Math.floor(utcOffsetAbsMinutes / 60))}:${pad2(utcOffsetAbsMinutes % 60)}`;
```